2-transform-data/bq_ingestion/extract_msgs_test.py

```diff
@@ -725,9 +726,51 @@ def test_convert_msg_to_json(self):
             got_json = em.convert_msg_to_json(test["msg"])
             self.assertEqual(want_json[key], got_json, "Convert message to json error")
 
+    
+    @mock.patch("google.cloud.storage.Client")
+    def create_client_mock(self, Client):
```

Comment:
pass in any variables you want to use in the mock before the objects you pass in. So if you want to simulate the table id you create, pass in the variable like `(self, table_id, Client)`